go/nogo.rst

```diff
@@ -86,6 +86,39 @@ even if the target is imported from an external repository. However, ``nogo``
 will not run when targets from the current repository are imported into other
 workspaces and built there.
 
+If you'd like to run all analyzers from ``golang.org/x/tools``, you can add a
+dependency on ``TOOLS_NOGO``.
```

Comment:
"add a dependency on" doesn't seem quite right, since it's a list of dependencies.

Perhaps instead, switch the order of the two examples, since using `tools_nogo` alone will be more common.

To run all the `golang.org/x/tools` analyzers, use `@io_bazel_rules_go//:tools_nogo`.

...

To run the analyzers from `tools_nogo` together with your own analyzers, use the `TOOLS_NOGO` list of dependencies.

...